sources/assets/zsh/zshrc

```diff
@@ -40,6 +40,9 @@ setopt HIST_FIND_NO_DUPS
 # Adding exegol bins to PATH
 export PATH=/opt/tools/bin:$PATH
 
+# Adding neovim to PATH
+export PATH=/opt/tools/nvim/bin:$PATH
```

Comment:
I see you have a `make install` command during the installation, why do you need to add a folder to the PATH ?
Even if the make install don't export correctly the tools to the system, please add a symlink in `/opt/tools/bin` instead of adding a new directory.